proposals/lip-0051.md

```diff
@@ -1859,10 +1858,6 @@ def transferCrossChain(
         emitFailedCrossChainTransferEvent(senderAddress, tokenID, amount, receivingChainID, recipientAddress, DATA_TOO_LONG)
         raise Exception("Data field too long.")
 
-    if Interoperability module is not registered:
```

Comment:
So when we remove this check, what actually happens if a custom module calls `transferCrossChain` when in the interoperability module is not registered? More precisely, what happens `Interoperability.send` in `transferCrossChainInternal` is called?

Reply:
it will fail, but it wont produce event

Reply:
Actually it will fail in `messageFeeTokenID = Interoperability.getMessageFeeTokenID(receivingChainID)` (line 1865); no event will be emitted and `transferCrossChainInternal` will not be called at all.

Reply:
ok. I think an additional comment that this function will fail if interoperability module is not registered would be nice, but it's not a must.

Reply:
I am not adding it, because there might be other places were a similar comment should be added, so prefer to skip it for consistency.